src/main/java/org/template/recommendation/PopModel.java

```diff
@@ -12,14 +13,13 @@
 import org.slf4j.LoggerFactory;
 import scala.Option;
 import scala.Tuple2;
-import scala.reflect.ClassTag;
-import scala.reflect.ClassTag$;
 
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
+@AllArgsConstructor
```

Comment:
From the link you pasted: 

> Static fields are skipped by these annotations.

Also, my Pop Model tests still work, and in them I don't pass anything related to logger into my PopModel constructors.